src/gameboy/main.cpp

```diff
@@ -86,7 +86,8 @@ int main(int argc, char *argv[]) {
 
 	auto cartridge = std::make_unique<Cartridge>(rom_path);
 	auto controller = std::make_unique<Controller>();
-	auto video = make_unique<Video>(controller.get());
+	auto video =
+	    make_unique<Video>(controller.get(), cartridge->meta_data->title);
```

Comment:
Can you pass the entire cartridge metadata object here? I know it's being passed by value but it's a lot easier for the video module to have all the information so if we need to pull something up on the Gui, it's there. You can store the metadata as a local member in the video module.

Reply:
Yeah sure, I don't think there's any issue in passing the entire cartridge in the main module, since it's only a pointer to the object. The thing is I forgot to change up the `main.cpp` since we are using `gameboy.cpp` as an executable for now.

Reply:
Ah, I meant only the Medatada object, not the cartridge file itself